src/main/java/store/bookshop/dto/order/CreateOrderRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package store.bookshop.dto.order;
+
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class CreateOrderRequestDto {
+    @NotNull
+    private String shippingAddress;
+
+    private List<CreateOrderItemRequestDto> orderItems;
```

Comment:
Delete this List.
You should take all cartItems from the ShoppingCart and create the Order - it is not FE job.